src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -845,9 +781,8 @@ def _set_fs_copy_params(self,
             self.fs_copy_cmd.set_fs_copy_params(
                 dst=str(dst_path))
         elif dst_type == "DAOS_UUID":
-            param = self._format_daos_path(dst_pool, dst_cont, dst_path)
             self.fs_copy_cmd.set_fs_copy_params(
-                dst=param)
+                dst=self.format_daos_path(dst_pool, dst_cont, dst_path))
```

Comment:
(pylint-no-member) Instance of 'DataMoverTestBase' has no 'format_daos_path' member